### ISSUE TITLE
Need to restore vc boot loader after windows update, but rescue disk says "not ready"

### ISSUE THREAD

**Issue by intently** [veracrypt/VeraCrypt#810]
Windows updated and changed my bootloader. This has happened before and I've been able to restore it from the rescue disk. However, this time it doesn't seem to be working. I only get a yellow "not ready" message when I attempt to "restore veracrypt loader to boot menu".

I'm also not able to "boot veracrypt loader from rescue disk".

Almost every option says "not ready".

Googling hasn't helped.

1.24-Update2 rescue disk

Help?

**Comment by eringiglio** [comment]
I am also dealing with the same issue. I haven't been able to figure out what to do. Did you ever come up with a solution?

**Comment by intently** [comment]
Yeah, it was related to UEFI. The problem was resolved when I removed my
SSD and re-installed it.

**Comment by eringiglio** [comment]
Rats--I'm getting this error on the whole machine. Did you just physically remove the drive and reinstall it? I'm not great at UEFI and BIOS issues generally, and absolutely wouldn't have installed VeraCrypt if I had realized that Windows 10 updating automatically could brick my machine likethis!

**Comment by intently** [comment]
Yes, physically remove the SSD and replace it, then let your BIOS detect it
again. Then try booting from the veracrypt boot sector (or w/e it's called)
on the ssd

**Comment by 75h7** [comment]
I had this same issue, something went wrong with my computer and it wouldn't boot any more - the bootloader wouldn't even load. I tried every option on the backup disk and most said "not ready" ("Decrypt OS" did something, completing after a few hours, but it still would not boot after). Nothing worked, including the recommendations in this thread. I eventually had to restore a months-old backup from before I encrypted my drive with veracrypt.
I did everything right: I read the manual and wizard instructions thoroughly, I made and verified the backup disk, I backed up a system image daily, I backed up my user files 3 times a day. My trust in this project has been trampled.
